src/apps/stunclient/stunclient.c

```diff
@@ -415,6 +421,25 @@ int main(int argc, char **argv)
   int c=0;
   int forceRfc5780 = 0;
 
+#if defined(WINDOWS)
```

Comment:
This is repeated multiple times - should be extracted to a function to avoid code repetition and errors